warehouse/core/src/main/java/datawave/attribute/EventField.java

```diff
@@ -0,0 +1,41 @@
+package datawave.attribute;
+
+import datawave.util.StringUtils;
+
+/**
+ * FIELD_NAME.COMMONALITY.GROUPING_CONTEXT
```

Comment:
FIELD_NAME.GROUP{.intermediate groups}.SUBGROUP
A little more to explain that this class cares about the first group and the last subgroup.

Reply:
Sounds good